src/Symfony/Component/ClassLoader/Tests/ApcClassLoaderTest.php

```diff
@@ -20,6 +20,8 @@ protected function setUp()
     {
         if (!(ini_get('apc.enabled') && ini_get('apc.enable_cli'))) {
             $this->markTestSkipped('The apc extension is not enabled.');
+        } elseif (defined('HHVM_VERSION') && !function_exists('apcu_clear_cache')) {
```

Comment:
you don't need this since apcu_clear_cache will always be provided by the polyfill thanks to require-dev